Examples/Framework/include/ACTFW/Validation/FakeRatePlotTool.hpp

```diff
@@ -26,22 +26,25 @@ class FakeRatePlotTool {
     std::map<std::string, PlotHelpers::Binning> varBinning = {
         {"Eta", PlotHelpers::Binning("#eta", 40, -4, 4)},
         {"Phi", PlotHelpers::Binning("#phi", 100, -3.15, 3.15)},
-        {"Pt", PlotHelpers::Binning("pT [GeV/c]", 20, 0, 100)},
+        {"Pt", PlotHelpers::Binning("pT [GeV/c]", 40, 0, 100)},
         {"Num", PlotHelpers::Binning("N", 30, -0.5, 29.5)}};
   };
```

Comment:
This would probably then profit a lot from moving to a `response-file` and feed it with options, so one can change this at runtime. The corresponding PR #114 has gone in already - so we could adapt this.

Reply:
That's true. I will open another PR for handling this after we are done with the CKF example, OK?